lib/src/api/community_moderation.dart

```diff
@@ -4,11 +4,66 @@ import 'package:interstellar/src/models/community.dart';
 import 'package:interstellar/src/utils/models.dart';
 import 'package:interstellar/src/utils/utils.dart';
 
+enum ReportStatus { any, appeal, approved, closed, pending, rejected }
```

Comment:
I haven't even heard of some of these report types while using Mbin. I know you can approve and reject reports with the Mbin web UI, but I don't think there's a way to actually appeal a report is there? Correct me if I'm wrong, but I feel like we should be able to simplify this by ignoring the `appeal` and `closed` status. The Mbin web UI, for example, only has filters for pending, approved, rejected, and all.

![](https://github.com/user-attachments/assets/bb7664da-a26a-47b8-960d-2aa19dee9bdc)